src/khoj/interface/web/chat.html

```diff
@@ -1372,7 +1557,7 @@
         }
         #input-row {
             display: grid;
-            grid-template-columns: auto 32px 40px;
+            grid-template-columns: auto 32px 40px 32px;
```

Comment:
Should we move the file upload button to the left of the chat input text field? Idea being that buttons on the right side are the more used actions like the current send message or voice note buttons (as closer to thumb on phone screen, so easier to click), the buttons on the left can be the less used actions

```suggestion
            grid-template-columns: 32px auto 32px 40px;
```